modules/apps/site-navigation/site-navigation-api/src/main/java/com/liferay/site/navigation/type/SiteNavigationMenuItemType.java

```diff
@@ -77,6 +77,13 @@ public default String getDisplayIcon(
 		return unicodeProperties.getProperty("displayIcon", StringPool.BLANK);
 	}
 
+	public default long getGroupId(
+		long companyId, long defaultGroupId,
```

Comment:
Is the `defaultGroupId` necessary? As far as I can tell it's always `0`.

Also, it feels a bit strange to have the implementations all be exactly the same. I'm not sure if there's a better way to handle it though.